src/components/tracked-controls.js

```diff
@@ -227,7 +229,10 @@ module.exports.Component = registerComponent('tracked-controls', {
     if (buttonState.touched === previousButtonState.touched) { return false; }
 
     evtName = buttonState.touched ? 'start' : 'end';
-    this.el.emit('touch' + evtName, {id: id, state: buttonState});
+    // Due to unfortunate name collision, add empty touches array to avoid Daydream error.
+    // this.el.emit doesn't allow changes to event outside detail...
+    // this.el.emit('touch' + evtName, {id: id, state: buttonState, touches: []});
+    this.el.dispatchEvent(new CustomEvent('touch' + evtName, {touches: [], detail: {id: id, state: buttonState}}));
```

Comment:
touchstart and touchend seem appropriate actually...

TouchPanner does not protect against unexpected input, but it is probably not the only one.  it might make sense to add empty touches to touchend as well, but that is not currently breaking anything